### PR TITLE
Fix case function description and example

### PR DESCRIPTION
**Pull request by DelazJ**
Imho, the current CASE ELSE description lets think that you get a result only when false.

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/expression.rst [hunk 1]
```diff
@@ -141,24 +141,25 @@ Conditionals
...
-                     depending on the conditional check
- regexp_match        Returns true if any part of a string matches the supplied
-                     regular expression
-===================  ==========================================================
+====================================  ===========================================================
+ Function                             Description
+====================================  ===========================================================
+ case when ... then ... end           Evaluates an expression and returns a result if true.
+                                      You can test multiple conditions
+ case when ... then .. else ... end   Evaluates an expression and returns a different result
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/expression.rst]: should it be better in uppercase?

**Comment by yjacolin** [comment]
LGTM as soon checks are finished

File: source/docs/user_manual/working_with_vector/expression.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/expression.rst]: I check and all the functions described are in lowercase, except few operators (with no reason, that said), reason why I changed. Too weird to have them lowercase? I prefer Uppercase or CamelCase, so I can rectify if you want. But for consistency...

Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/expression.rst]: I have no strong opinion but in QGIS "CASE WHEN ..." is in uppercase and usually SQL statement are in uppercase, they are not function. coalesce is a function so it should be kept in lowercase (as regexp_match).

Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/expression.rst]: Indeed. Fixed.

**Pull request merged**